test/controllers/GuidanceControllerSpec.scala

```diff
@@ -25,162 +25,206 @@ import play.api.test.FakeRequest
   }
 
-  "Calling the scratch process endpoint with a valid UUID and existing sessionId" should {
 
-    trait ScratchTest extends MockGuidanceService {
-      lazy val uuid = "683d9aa0-2a0e-4e28-9ac8-65ce453d2730"
-      lazy val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("GET", "/").withSession(SessionKeys.sessionId -> "SESSIONID")
-      val expectedUrl = "/start-url"
+  "Calling the scratch process endpoint with a valid UUID and existing sessionId" should {
 
+    "Use the existing session ID" in new ScratchTest {
       MockGuidanceService
         .scratchProcess(uuid, uuid)
         .returns(Future.successful(Some(expectedUrl)))
 
-      private lazy val errorHandler = app.injector.instanceOf[config.ErrorHandler]
-      private lazy val view = app.injector.instanceOf[views.html.render_page]
-
-      lazy val target = new GuidanceController(errorHandler, view, mockGuidanceService, stubMessagesControllerComponents())
-      lazy val result: Future[Result] = target.scratch(uuid)(fakeRequest)
-    }
-
-    "Use the existing session ID" in new ScratchTest {
+      val result: Future[Result] = target.scratch(uuid)(fakeRequest.withSession(SessionKeys.sessionId -> "SESSIONID"))
       session(result).data.get(SessionKeys.sessionId) mustBe Some("SESSIONID")
     }
   }
 
   "Calling the scratch process endpoint with an invalid UUID" should {
 
-    trait ScratchTest extends MockGuidanceService {
-      lazy val unknownUuid = "683d9aa0-2a0e-4e28-9ac8-65ce453d2731"
-      lazy val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("GET", "/")
-      val expectedUrl = "/start-url"
-
+    "return a NOT_FOUND error" in new ScratchTest {
       MockGuidanceService
-        .scratchProcess(unknownUuid, unknownUuid)
+        .scratchProcess(uuid, uuid)
         .returns(Future.successful(None))
+      val result: Future[Result] = target.scratch(uuid)(fakeRequest)
+      status(result) mustBe Status.NOT_FOUND
+    }
 
-      private lazy val errorHandler = app.injector.instanceOf[config.ErrorHandler]
-      private lazy val view = app.injector.instanceOf[views.html.render_page]
+  }
 
-      lazy val target = new GuidanceController(errorHandler, view, mockGuidanceService, stubMessagesControllerComponents())
-      lazy val result: Future[Result] = target.scratch(unknownUuid)(fakeRequest)
-    }
+  trait StartJourneyTest extends MockGuidanceService {
+    lazy val processId = "ext90002"
+    lazy val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("GET", "/")
+    val expectedUrl = "/start-url"
 
-    "return a NOT_FOUND error" in new ScratchTest {
-      status(result) mustBe Status.NOT_FOUND
-    }
 
+    private lazy val errorHandler = app.injector.instanceOf[config.ErrorHandler]
+    private lazy val view = app.injector.instanceOf[views.html.standard_page]
+    private lazy val questionView = app.injector.instanceOf[views.html.question_page]
+
+    lazy val target = new GuidanceController(errorHandler, 
```

Comment:
Again IntelliJ is reporting trailing whitespace.

Reply:
will clear